app/javascript/components/App.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import Greeting from "./greeting";
+const App = ()=>{
+
+    return < Greeting />
+}
+
```

Comment:
- [ ] It will be great to ensure that you have routes created by react-router. This is a major requirement for this project. Instead of just displaying the  `Greeting` component, there should be a route created for it in the app.js component